Airports/EGSS/Fixes.txt

```diff
@@ -17,3 +17,68 @@ SSS18 N051.40.45.120 E000.18.31.920
 SSW01 N051.51.46.360 E000.12.05.900
 TOTVO N052.01.18.820 E000.26.27.790
 XIGAR N051.48.50.000 E000.15.46.000
+
+; Air Network Fixes
+; VFR Management
+SSAUDL N052.00.15.000 E000.12.25.000
+SSBERN N051.45.49.400 E000.18.10.790
+SSBRAI N051.52.42.000 E000.33.14.000
+SSCANF N051.52.13.720 E000.19.23.170
+SSCHEL N051.44.00.000 E000.28.24.000
+SSCHIP N051.42.45.560 E000.14.42.020
+SSDEBD N051.59.31.050 E000.16.29.380
+SSDIAM N051.52.40.000 E000.14.09.000
+SSEPPI N051.42.00.000 E000.06.40.000
+SSHAVE N052.04.57.000 E000.26.04.000
+SSHAZE N051.53.38.880 E000.10.15.050
+SSNUTH N051.59.24.000 E000.03.43.000
+SSSPRI N051.58.02.630 E000.22.08.490
+SSWARE N051.48.42.000 E000.01.36.000
+SSPUCK N051.53.06.000 E000.00.16.000
+
+; Circuits
+SS04UC N051.54.56.369 E000.16.47.013
+SS04CD N051.52.57.806 E000.20.52.072
+SS04DB N051.49.21.058 E000.14.59.125
+SS04BF N051.51.6.653 E000.12.06.505
```

Comment:
```suggestion
SS04BF N051.51.06.653 E000.12.06.505
```
All co-ordinates should ideally have 10 lattitude + 10 longitude numbers. Presumably this is 06?

Reply:
I would guess 60 (as 0.60 = 0.6) @craigstewart4717 I dont suppose you can try out locally and push a change?

Reply:
@1adamf I think it's done as degrees minutes seconds decimal-seconds (hence the 3 .s) so I wouldn't expect to ever have a 60 there as that would presumably roll over into the minutes, although I could be wrong so do check @craigstewart4717

Reply:
just to confirm Peter you are correct in the lat long format.